src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -122,8 +122,8 @@ private ActionProcessingService getActionProcessingService(Action action) {
     ActionCase actionCase = actionCaseRepo.findById(action.getCaseId());
 
     if (actionCase == null) {
-      log.with("action", action).error("Cannot find case for action");
-      throw new IllegalStateException();
+      log.with("action", action).info("Case no longer exists for action");
+      action.setState(ActionState.ABORTED);
```

Comment:
This bypasses the state transition stuff. Are we OK to do that?

Reply:
also, there's no save to persist the change to the DB

Reply:
Good shout completely spaced on that. Changing to work with transitions